python/lsst/ts/gis/commander.py

```diff
@@ -138,18 +146,14 @@ def get_raw_string(self, reply):
         raw_status : `bytearray`
             The unfiltered response of the reply in a byte array.
         """
-        raw_status = bytearray()
+        bit_status = ""
```

Comment:
I think these next lines until the return may be simplified as

```
bit_status = []
try:
    for data in reply.registers:
        raw_status = data.to_bytes(2, sys.byteorder)
        bit_status.append(bin(int.from_bytes(raw_status))[2:].zfill(16))
    self.log.debug(f"{bit_status=}")
    return " ".join(bit_status)
```

Reply:
Edit, maybe better not to convert this to a string at all but simply return the array. That will help with my comment in `component.py`.